docs/tsconfig.json

```diff
@@ -31,7 +31,7 @@
     "incremental": true
   },
   "include": ["next-env.d.ts", "**/*.ts", "**/*.tsx"],
-  "exclude": ["node_modules", "./cypress.config.ts"],
+  "exclude": ["node_modules", "./cypress.config.ts", "./cypress/e2e/*.ts"],
```

Comment:
This would prevent the tests from being typechecked? Should we come back to this one?

Reply:
The only test we have currently is the link checker which is being reworked. To run ts on it rn would mean to add `"types":["cypress"]` to tsconfig file, which makes cypress and jest types clash. Since the link checker is being refactored to remove cypress, was thinking to defer addressing this to later [will add to ticket].

Reply:
MSTM

Reply:
Added to backlog ticket.